example/styles.css

```diff
@@ -29,4 +29,20 @@ z-index: 10; }
   padding-bottom: 25px;
   margin-right: 65px;
   display: inline;
+}
+
+.badge {
```

Comment:
Hey, the PR is all set. Only thing is that i think we should move this CSS to /src folder and on grunt build -> we should copy this from /src to /dist folder if it not already there. Makes sense? Thanks

Reply:
Okay. Shall work on it. Thanks!